doc/source/dem_class.md

```diff
@@ -157,10 +168,11 @@ stable terrain as a proxy.
 
 ```{code-cell} ipython3
 # Estimate elevation uncertainty assuming both DEMs have similar precision
-sig_dem, rho_sig = dem.estimate_uncertainty(dem_tba_coreg, precision_of_other="same")
+sig_dem, rho_sig = dem.estimate_uncertainty(dem_tba_coreg, precision_of_other="same", random_state=42)
```

Comment:
I was also not aware of this method!!
Maybe explain in the comment why random_state is used and that it should not be used in the general case.

Reply:
Done!